### PR TITLE
Moving around ItemsRepeater samples to organize them better 

### PR DESCRIPTION
**Pull request by ranjeshj**
Moving around ItemsRepeater samples and organizing them a little better. This is in preparation for adding a handful of other samples.

### CODE REVIEW

File: dev/Repeater/TestUI/Samples/LayoutSamples/VirtualLayoutPages/VirtualizingUniformStackLayoutSamplePage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            if(!int.TryParse(indexTb.Text, out index))
+            {
+                index = 0;
+            }
+
+            int count = 1;
+            if(!int.TryParse(countTb.Text, out count))
+            {
+                count = 1;
+            }
```
[the remark this answers is not in the document]
Comment (ranjeshj) [reply dev/Repeater/TestUI/Samples/LayoutSamples/VirtualLayoutPages/VirtualizingUniformStackLayoutSamplePage.xaml.cs]: Could be used but this is fine for a test sample so I'm going to leave it as is.

File: dev/Repeater/TestUI/Samples/LayoutSamples/VirtualLayoutPages/PinterestLayoutSamplePage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                           }));
+
+            repeater.ItemsSource = data;
+            bringIntoView.Click += BringIntoView_Click;
+        }
+
+        private void BringIntoView_Click(object sender, RoutedEventArgs e)
+        {
+            int index = 0;
+            if (int.TryParse(tb.Text, out index))
```
[the remark this answers is not in the document]
Comment (ranjeshj) [reply dev/Repeater/TestUI/Samples/LayoutSamples/VirtualLayoutPages/PinterestLayoutSamplePage.xaml.cs]: updated to textbox. The intention is to allow bringing any index into view.

File: dev/Repeater/TestUI/Samples/LayoutSamples/VirtualLayoutPages/VirtualizingStackLayoutSamplePage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                           }));
+
+            repeater.ItemsSource = data;
+            bringIntoView.Click += BringIntoView_Click;
+        }
+
+        private void BringIntoView_Click(object sender, RoutedEventArgs e)
+        {
+            int index = 0;
+            if (int.TryParse(tb.Text, out index))
```
[the remark this answers is not in the document]
Comment (ranjeshj) [reply dev/Repeater/TestUI/Samples/LayoutSamples/VirtualLayoutPages/VirtualizingStackLayoutSamplePage.xaml.cs]: updated to textbox

File: dev/Repeater/TestUI/RepeaterTestUIPage.xaml [hunk 4]
```diff
@@ -5,89 +5,87 @@
...
             <controls:StackLayout x:Key="stack"  Spacing="5" />
-        </StackPanel.Resources>
+        </controls:LayoutPanel.Resources>
 
-        <controls:LayoutPanel >
-            <controls:LayoutPanel.Layout>
-                <controls:FlowLayout Orientation="Vertical" MinColumnSpacing="5" MinRowSpacing="5"/>
-            </controls:LayoutPanel.Layout>
+        <StackPanel>
+            <TextBlock Text="ItemsSource Samples" Style="{ThemeResource HeaderTextBlockStyle}"/>
```
Comment (chingucoding) [inline dev/Repeater/TestUI/RepeaterTestUIPage.xaml]: nit: You could also use the `StandardGroupHeader` style that was introduced with #1793, but this looks fine to me.

**Pull request merged**